_targets.R

```diff
@@ -14,80 +11,8 @@ parameterCd <- '00010'
 startDate <- "2014-05-01"
 endDate <- "2015-05-01"
```

Comment:
As your output shows, the pipeline runs fine w/ these parameters defined in the main `_targets.R` file, but typically we'd define these phase-specific parameters within the phase-specific .R files where the targets for each phase are defined.

Reply:
That makes a lot of sense! All taken care of now.